### ISSUE TITLE
[Feature request]  Adding the 'src'  attribute to the script tag and make loading markdown from a .md file possible

### ISSUE THREAD

**Issue by salah-saleh** [PolymerElements/marked-element#34]
I wish if something like the code below would be possible. I think the element would be much more usable.

```
<marked-element>
      <div class="markdown-html"></div>
      <script type="text/markdown" src="/data/data.md"></script>
</marked-element>
```

**Comment by jroosing** [comment]
I need this. I am currently porting zero-md to polymer 2 cause it more closely resembles my needs (loading markdown from external source)

**Comment by FredKSchott** [comment]
The team doesn't have the resources to add this right now, but I'd happily work with anyone willing to create a PR to add this.

**Comment by stramel** [comment]
@FredKSchott If you don't mind highlighting how you feel it should be gone about I will submit a PR. It sounds like I would need to add an iron-ajax to make a request if the src attribute is present?

**Comment by jroosing** [comment]
Allright @FredKSchott thanks for the info!

@stramel you could use iron-ajax, but I generally prefer to keep elements as flat as possible, so in this case you could just use plain XHR. @FredKSchott What is your stance on this?

**Comment by FredKSchott** [comment]
I don't have a strong stance for or against `<iron-ajax>`, but since we're talking about an optional feature I agree that adding a dependency to everyone isn't great. So I'd prefer to see native XHR if the request is simple.

**Comment by stramel** [comment]
@fredkschott @joeyroosing I will take a stab at it!

**Comment by stramel** [comment]
@salah-saleh @joeyroosing This has been merged and tagged as v1.4.0.
